docs/articles/new-expensify/bank-accounts-and-credit-cards/Connect-a-Bank-Account.md

```diff
@@ -1,26 +1,26 @@
 ---
-title: Business Bank Accounts - USD
-description: How to add/remove Business Bank Accounts (US) 
+title: Connect a Business Bank Account - US
+description: How to connect a business bank account to Expensify (US) 
 ---
 # Overview
 Adding a verified business bank account unlocks a myriad of features and automation in Expensify. 
 Once you connect your business bank account, you can:
-- Pay employee expense reports via direct deposit (US)
-- Settle company bills via direct transfer
-- Accept invoice payments through direct transfer
-- Access the Expensify Card
+- Reimburse expenses via direct bank transfer
+- Pay bills
+- Collect invoice payments
+- Issue the Expensify Corporate Card
```

Comment:
I think we should leave this as "Expensify Card", not "Expensify Corporate Card" as that's not the title!